### ISSUE TITLE
TypeError: unsupported operand type(s) for /: 'str' and 'str' in language_model_trainer.py

### ISSUE THREAD

**Issue by xuxingya** [zalandoresearch/flair#340]
An error occured when I try to train my own embedding following Tutorial 9. My system is Ubuntu 18 and python3.6.
```
corpus = TextCorpus('/data/resume_extract_file/corpus',
                    dictionary,
                    is_forward_lm,
                    character_level=True)
```
![image](https://user-images.githubusercontent.com/13343428/50547009-9fe38480-0c6c-11e9-8bab-ffcbca7a73d1.png)
It looks like the problem was occurred by the 24 line of flair/flair/trainers/language_model_trainer.py
self.train_path = path / 'train'.

**Comment by stefan-it** [comment]
Should be:

```python
from pathlib import Path

corpus = TextCorpus(Path('/data/resume_extract_file/corpus'),
                    dictionary,
                    is_forward_lm,
                    character_level=True)
```

**Comment by stefan-it** [comment]
I made a PR to correct this in the documentation :)

**Comment by xuxingya** [comment]
Thank you for you fast respond :)

**Comment by alanakbik** [comment]
Ah, great! @xuxingya thanks for spotting this! @stefan-it thanks for fixing this :)

**Comment by alanakbik** [comment]
fixed in #341
